src/goto-symex/README.md

```diff
@@ -276,3 +276,10 @@ digraph G {
   2 -> 3 [label="counter-examples"];
 }
 \enddot
+
+\section Concurrency
+
+\subsection Partial Order Reduction
```

Comment:
I'd name it just `Partial Order Concurrency` as it's not really partial order reduction.

Reply:
You mean that its used to implement memory models but the ordering is never used for reduction?